modules/contentbox/modules/contentbox-admin/handlers/authors.cfc

```diff
@@ -311,7 +311,13 @@ component extends="baseHandler"{
      * Show the two-factor authentication screen for forced enrollment
      */
     function forceTwoFactorEnrollment( event, rc, prc ) {
-        prc.author = prc.oCurrentAuthor;
+		if( flash.exists( "authorData" ) ){
+			// Inflate author for requested events
+			prc.author = authorService.get( flash.get( "authorData" ).authorID );
```

Comment:
@gpickin is this the  risk that you were saying?

Reply:
No because this is only loading if we see it in the flash.
There is another part of the flow that just looks for the authorid

moving it should fix that issue, i think.